staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -545,6 +551,113 @@ message ListPodSandboxResponse {
     repeated PodSandbox items = 1;
 }
 
+message PodSandboxStatsRequest {
```

Comment:
I assume proto changes will not be included in this PR but rather in #102789 instead correct?

Reply:
yeah this is based on that